### PR TITLE
Support `exclude-elements` option

### PR DESCRIPTION
**Pull request by KazuyaHayashi**
Ref #28
Support `exclude-elements` option on `diff`, `changelog` and `breaking` action.

### CODE REVIEW

**Comment by effoeffi** [comment]
Thank you for your contribution @KazuyaHayashi. I recommend testing the output of the 'diff' action. Currently, the 'diff' action does not provide any output. I suggest creating another pull request to include output in the 'diff' action, and then we can test the results using 'exclude-elements'.

File: specs/base-exclude-elements.yaml [hunk 1]
```diff
@@ -0,0 +1,111 @@
+openapi: "3.0.0"
```
Comment (effoeffi) [inline specs/base-exclude-elements.yaml:1]: Can we use `revision.yaml`?

Comment (KazuyaHayashi) [reply specs/base-exclude-elements.yaml:1]: @effoeffi Thank you for your comments!

`revision.yaml` doesn't contain any changes that can be ignored with `--exclude-elements` option, so I created `base-exclude-elements.yaml` from `base.yaml`.

Run oasdiff without `--exclude-elements`
```docker run --rm -t tufin/oasdiff diff --format text https://raw.githubusercontent.com/oasdiff/oasdiff-action/main/specs/base.yaml https://raw.githubusercontent.com/oasdiff/oasdiff-action/main/specs/revision.yaml
### New Endpoints: 1
--------------------
GET /healthcheck  

### Deleted Endpoints: None
---------------------------

### Modified Endpoints: None
----------------------------
```

Run oasdiff with `--exclude-elements`
```
docker run --rm -t tufin/oasdiff diff --format text --exclude-elements description,title,summary https://raw.githubusercontent.com/oasdiff/oasdiff-action/main/specs/base.yaml https://raw.githubusercontent.com/oasdiff/oasdiff-action/main/specs/revision.yaml
### New Endpoints: 1
--------------------
GET /healthcheck  

### Deleted Endpoints: None
---------------------------

### Modified Endpoints: None
----------------------------
```

**Comment by KazuyaHayashi** [comment]
@effoeffi I modified the test case and opened new PR. Also I closed this PR. Could I have your review? Thanks!
https://github.com/oasdiff/oasdiff-action/pull/33

**Pull request closed without merging**